src/main/java/com/ss/training/lms/controller/AdminPublisherController.java

```diff
@@ -39,4 +40,21 @@
 		return new ResponseEntity<Publisher>(publisher, status);
 	}
 	
+	/**
+	 * @return
+	 */
+	@RequestMapping(path = "/lms/admin/publisher")

```

Comment:
Path should be publishers. instead of publisher

Reply:
I thought Akbar said the plural form should only be used when followed by an identifier to select one or more items out of a collection.